bindings/rascal/neighbourlist/base.py

```diff
@@ -11,28 +11,53 @@
                                  '').replace('make_structure_manager_', '')
         _neighbourlists[name] = v
 
+_structure_collections = {}
+for k, v in NeighbourList.__dict__.items():
+    if "ManagerCollection" in k:
+        name = k.lower().replace('managercollection_', '')
+        _structure_collections[name] = v
 
 def NeighbourListFactory(nl_options):
     names = []
-    args = []
+    kargs = []
```

Comment:
This naming change doesn't make it any clearer what this thing is -- what's `k` and what does it have to do with `args`?

Reply:
managers need arguments to be initialized right ? here they are then